datarobot_provider/example_dags/datarobot_azure_storage_batch_scoring_dag.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+Example of Aiflow DAG for DataRobot Batch Scoring using Azure Blob Storage as source and destination,
+using a preconfigured "DataRobot Azure Storage Credentials" from Airflow Connection.
+DataRobot Azure Storage Credentials can be configured using Airflow UI (Admin->Connections) or Airflow API
+Config example for this dag:
+{
+    "azure_storage_credentials": "demo_azure_storage_test_credentials",
+    "deployment_id": "put_your_deployment_id",  # you can set deployment_id here
+    "score_settings": {
+        "intake_settings": {
+            "type": "azure",
+            "url": "https:// ... .blob.core.windows.net/.../input_filename.csv",
+        },
+        "output_settings": {
+            "type": "azure",
+            "url": "https:// ... .blob.core.windows.net/.../output_filename.csv",
+        },
+        # If passthrough columns are required, use this line:
+        "passthrough_columns": ['column1', 'column2'],
+    },
+}
+"""
+from datetime import datetime
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.credentials import GetCredentialIdOperator
+from datarobot_provider.operators.datarobot import ScorePredictionsOperator
+from datarobot_provider.sensors.datarobot import ScoringCompleteSensor
+
+
+@dag(
+    schedule_interval=None,
+    start_date=datetime(2023, 1, 1),
+    tags=['example', 'azure'],
+    params={
+        "azure_storage_credentials": "demo_azure_storage_test_credentials",
+        "deployment_id": "put_your_deployment_id",  # you can set deployment_id here
+        "score_settings": {
+            "intake_settings": {
+                "type": "azure",
+                "url": "https:// ... .blob.core.windows.net/.../input_filename.csv",
+            },
+            "output_settings": {
+                "type": "azure",
+                "url": "https:// ... .blob.core.windows.net/.../output_filename.csv",
+            },
+        },
+    },
+)
+def datarobot_azure_storage_batch_scoring(deployment_id="put_your_deployment_id"):
```

Comment:
```suggestion
def datarobot_azure_storage_batch_scoring(deployment_id=None):
```
ditto about `None` here?